pkgs/build-support/setup-hooks/audit-tmpdir.sh

```diff
@@ -15,25 +15,56 @@ auditTmpdir() {
 
     echo "checking for references to $TMPDIR/ in $dir..."
 
-    local i
-    find "$dir" -type f -print0 | while IFS= read -r -d $'\0' i; do
-        if [[ "$i" =~ .build-id ]]; then continue; fi
+    local tmpdir="$(mktemp -d)"
```

Comment:
I believe she'll check usually complaints when local var declarations are in one line with command substitution. This is what it usually suggests:

```
local foo
foo=$(bar)
```